kubejs/server_scripts/base/featrues/devices.js

```diff
@@ -72,6 +72,40 @@ if (
         removeRecipe({ id: "create:crafting/kinetics/controller_rail" });
       },
     },
+    {
+      tag: "forge:device/furniture",
+      generic: "ptdye:furnished_device",
+      base: "minecraft:stick",
+      incomplete: "ptdye:incomplete_furnished_device",
+      assembly: [
+        "#minecraft:wool_carpets",
+        "#forge:nuggets",
+        "#minecraft:planks",
+        "#minecraft:planks"
+      ],
+      included_devices: [
+        'create:cuckoo_clock', 'minecraft:bookshelf', 'minecraft:note_block', 'minecraft:oak_door', 'minecraft:oak_sign',
+        'minecraft:oak_button', 'minecraft:composter', 'supplementaries:hanging_sign_oak', 'supplementaries:pulley_block', 'minecraft:barrel',
+        'minecraft:oak_pressure_plate', 'create:wooden_bracket', 'supplementaries:speaker_block', 'minecraft:painting', 'supplementaries:notice_board',
+        'supplementaries:bellows', 'minecraft:light_blue_bed', 'minecraft:chest', 'supplementaries:sign_post_oak', 'minecraft:item_frame',
+        'storagedrawers:oak_full_drawers_1', 'minecraft:jukebox', 'create:white_seat', 'minecraft:campfire', 'minecraft:lectern'
+      ],
+      amount_crafted: 4,
+      post_logic: () => {
+        ServerEvents.blockLootTables(event=>{
```

Comment:
Should probably move to a function that will delegate it for a centrilized event like we do for recipes. Not important now though, for now this is good